### DISCUSSION TITLE
HtmlContentEditor is not working when opened in Panel

### DISCUSSION THREAD

**Discussion by EdwinThomas11** [serenity-is/Serenity#7008] [Help]
I tried to utilize the HtmlContentEditor within the form. It functions properly when the form is displayed as a dialog, but it doesn't work when the form is displayed as a panel.

Appreciate the inputs provided.

**Comment by volkanceylan** [comment]
I don't see any issue here:

<img width="1130" alt="image" src="https://github.com/serenity-is/Serenity/assets/752470/92f1c5cb-7831-4058-a9d4-257dd2d3bb86">
